### PR TITLE
[Docs] Update OpenBB Copilot docs

### PR DESCRIPTION
**Pull request by mnicstruwig**
# Pull Request OpenBB

Please go the the `Preview` tab and select the appropriate PR sub-template:

* [OpenBB Platform](?expand=1&template=platform_pull_request_template.md)
* [OpenBB Terminal](?expand=1&template=terminal_pull_request_template.md)
* [OpenBB Developers](?expand=1&template=obb_developer_pull_request_template.md)

Overview
https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/6466c3e3-111b-4246-ae9b-b6403712c862

Chat history
https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/312510fb-fb17-474c-8698-a2960134e285

Clearing chat history

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/dd67030e-bfd4-4e8c-b9f4-e67d3dbd2249

Chatting with dashboard

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/0c502fa9-dae3-45f1-996d-2b9940161c24

Citations

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/2156f1cb-f49e-4481-8131-41fcd6b91672

Querying widgets

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/1335e310-cd65-4917-bc34-1de8b3e5f7fc

Chat with files

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/905eb674-5619-4797-8adf-9cf13a846792

Add your own copilot

https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/15d2d827-715e-42f3-be62-b3c7f8b26fda

### CODE REVIEW

**Comment by mnicstruwig** [comment]
https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/71ea559b-8b76-47b6-9fd5-4c9753a7747c

**Comment by mnicstruwig** [comment]
Still need to create a separate page for the "Bring your own copilot" feature.

File: website/content/pro/copilot/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+To open OpenBB Copilot, click on the icon located at the bottom right of the
+Terminal Pro screen.
+
+You can interact with OpenBB Copilot either through typing text or using your
+voice.  Under most circumstances, you can treat OpenBB Copilot like your very
+own personal research assistant. OpenBB Copilot can answer general financial
+questions, use the data displayed in the dashboard to perform analytical tasks,
+and even interact with files that you choose to upload.
+
+We'll be exploring each of these features the following sections below.
```
Comment (jmaslek) [inline website/content/pro/copilot/index.md:37]: missing a word: each of these features in the following sections.

File: website/content/pro/copilot/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+## Understanding chat history
+
+<ReactPlayer width="50%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/312510fb-fb17-474c-8698-a2960134e285' />
+
+OpenBB Copilot is a conversational agent. This means that OpenBB Copilot uses
+previous messages and answers in the current chat to help answer your query.
+This allows you to guide OpenBB Copilot to perform the task you want, or ask
+follow-up questions.
+
+In the example above, the user first asked about the Price-to-earnings (P/E)
```
Comment (jmaslek) [inline website/content/pro/copilot/index.md:48]: capitzlization on Price-to-earnings

File: website/content/pro/copilot/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+uploaded files, if necessary, to answer your queries. If OpenBB Copilot uses the data
+contained in your files to answer a query, it will cite which files it used (and
+in the case of PDFs, the specific page).
+
+### A note on filenames
+OpenBB Copilot makes use of filenames to assess whether a file is
+relevant to the user's query. As a result, it is highly recommended that you use
+filenames that are descriptive of the data that they contain. For example, given
+a PDF file containing a technical report from TSLA released in 2024, a good
+filename would be `tsla_technical_report_2024.pdf`.
```
Comment (jmaslek) [inline website/content/pro/copilot/index.md:139]: not a complaint or blocking, but we could put this in one of those fun info boxes

File: website/content/pro/copilot/bring-your-own-copilot.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<ReactPlayer width="70%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/15d2d827-715e-42f3-be62-b3c7f8b26fda' />
+
+After deploying your custom Copilot, you can add it to Terminal Pro.  To do
+this, navigate to the "Add copilot" section and follow the prompts:
+
+- Click on the "Add copilot" button.
+- Enter the API endpoint of your custom copilot.
+- Confirm the addition, and your custom Copilot will be integrated and ready for use.
+ 
+
```
Comment (jmaslek) [inline website/content/pro/copilot/bring-your-own-copilot.md:49]: Can add some final generic statements like "Then you are ready to go" or some jargon

**Review by jmaslek — APPROVED** [review]
Couple small nits but lgtm

File: website/content/pro/copilot/index.md (hunk 2 above)
Comment (mnicstruwig) [reply website/content/pro/copilot/index.md:48]: > Capitalization of sub-headers, "Understanding Chat History"

I'm following the pattern set in the Terminal Pro section of the docs, which doesn't use Title Case for subheadings.

File: website/content/pro/copilot/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+In the example above, the user first asked about the Price-to-earnings (P/E)
+ratio. After OpenBB Copilot answered the question, the user asked a
+follow-up question regarding additional ratios that may be important. Since
+OpenBB Copilot is aware of the context of the current conversation, it
+proceeded to answer the user's query within the context of the conversation
+(in this case, financial ratios) by suggesting other important financial ratios that might be
+of use.
+
+If you'd like to clear the history of the current conversation, you can do so by
+clicking on the bin icon in the OpenBB Copilot chatbox. It is usually a good
```
[the remark this answers is not in the document]
Comment (mnicstruwig) [reply website/content/pro/copilot/index.md:57]: Switching to "trashcan".

**Pull request merged**